qwt/COPYING

```diff
@@ -16,7 +16,7 @@ exceptions:
        widgets against a user-supplied version of Qwt.
```

Comment:
Please revert all changes in qwt directory and set the full directory to the whitelist (never check it again), since we would like to be as close as possible to the original sources of qwt, such that future changes can be better analyzed by a diff command.